### ISSUE TITLE
[FR] add user to source config when viewing submissions

### ISSUE THREAD

**Issue by leevigraham** [verbb/formie#453]
**What are you trying to do?**

View userId and IP in submissions table

**What's your proposed solution?**

Add these options to the source config

![image](https://user-images.githubusercontent.com/25124/116832532-8e259980-abf8-11eb-9d26-8b13f89500dd.png)

**Comment by engram-design** [comment]
IP Address is in there are the moment, FYI. I'll get the User column added.

**Comment by leevigraham** [comment]
> IP Address is in there are the moment

Derp… I just added that to the comment without checking.

**Comment by engram-design** [comment]
Added for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.4](https://github.com/verbb/formie/releases/tag/1.4.4)
